types/test/usage.ts

```diff
@@ -15,6 +15,15 @@ async function test(): Promise<void> {
 
     user = await User.findOne();
 
+    if (!user) {
```

Comment:
This is required?

Reply:
Method findOne may return null.

Reply:
So typescript gives you warning/error without this?

Reply:
![Screenshot](https://user-images.githubusercontent.com/9392723/60522158-308e4080-9d34-11e9-9cdf-6f0e814e3a1c.PNG)

Reply:
@sushantdhiman fyi this will not error when specifyin` rejectOnEmpty`, TS is got the smarts :)

Reply:
:+1: